samples/change-notification/csharp/ChangeNotification/Helper/ChangeNotification.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using AdaptiveCards;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace ChangeNotification.Helper
+{
+    public class ChangeNotificationHelper
+    {
+
+        public Microsoft.Bot.Schema.Attachment ShowAdaptiveCard(string Header, ChangeNotification.Model.ResourceData InfoData)
```

Comment:
Could you please rename this as this method just returns the Adpative card? Can we rename it to GetAvailabilityChangeCard()?